{
  assertEquals("Get returned incorrect object","Test 80",tVector.get(80));
  tVector.add(25,null);
  assertNull("Returned incorrect element--wanted null",tVector.get(25));
  try {
    tVector.get(-5);
    fail("ArrayIndexOutOfBoundsException expected");
  }
 catch (  ArrayIndexOutOfBoundsException e) {
  }
  try {
    tVector.get(tVector.size() + 1);
    fail("ArrayIndexOutOfBoundsException expected");
  }
 catch (  ArrayIndexOutOfBoundsException e) {
  }
}
